Components/Input/Input.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./inputstyles.css" />
+    <title>Input Components</title>
+  </head>
+  <body>
+    <h1>Input Fields</h1>
+    <h2>Text Field</h2>
+
+    <div class="simple-textbox">
+      <div style="margin-top: 1rem">
```

Comment:
inline styles can be removed as it has the **highest specificity**, you might face issues styling your component in future.